core/domain/feedback_services.py

```diff
@@ -371,6 +397,85 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_thread_summaries(user_id, full_thread_ids):
+    """Returns a list of summaries corresponding to each of the threads given.
+
+    Args:
+        thread_ids: list(str). The ids of the threads excluding the exploration
+            id part.
+        exploration_ids: list(str). The ids of the explorations to which the
+            threads belong.
+
+    Returns:
+        list(dict). A list of dictionary containing the summaries of the threads
+            given to it.
+    """
+    exploration_ids = [thread_id.split('.')[0] for thread_id in full_thread_ids]
```

Comment:
Maybe useful if you want to do the split just once: https://stackoverflow.com/questions/7558908/unpacking-a-list-tuple-of-pairs-into-two-lists-tuples